test/unit/message_spec.lua

```diff
@@ -4,22 +4,28 @@ local ffi = helpers.ffi
 local eq = helpers.eq
 local to_cstr = helpers.to_cstr
 
-local message = helpers.cimport('./src/nvim/message.h')
+local message = helpers.cimport('./src/nvim/message.h', './src/nvim/memory.h',
```

Comment:
consider renaming this to `cimported` (we do this in some other tests). I think long ago cimport() didn't support multiple arguments or something, but anyways `message.xmalloc()` looks a bit weird, `cimported.xmalloc()` is a reasonable pattern.